Sources/AngouriMath/Core/Parser.cs

```diff
@@ -47,6 +50,19 @@ public static Entity Parse(string source)
                 AngouriMathLexer.DefaultVocabulary.GetDisplayName(token.Type) is var type
                 && type is not PARENTHESIS_OPEN && type.EndsWith("('") ? FUNCTION_OPEN : type;
 
+            CommonToken GetPowerToken(AngouriMathLexer lexer, string tokenType)
+            {
+                if (MathS.Settings.ExplicitParsingOnly && tokenType == VARIABLE)
+                {
+                    angouriMathTextWriter.WriteLine("Cannot power a number without '^' When SetExplicitParsingOnly(true)  has been called"+ $"\n"+ 
+                        "If you want to power a number without '^' Dont call SetExplicitParsingOnly(true) ");
+                }
+                return lexer.Power;
+            }
+            CommonToken GetMultiplyToken(AngouriMathLexer lexer )
+            {
+                return lexer.Multiply;
+            }
```

Comment:
May I wonder what is the point of this function? Why not just to keep `lexer.Multiply` where necessary?